--- hp-deskjet_f4100_series.ppd.orig	2024-04-25 01:00:15.077900525 -0500
+++ hp-deskjet_f4100_series.ppd	2024-04-25 00:59:24.741902159 -0500
@@ -41,9 +41,9 @@
 *cupsModelNumber: 0
 *cupsManualCopies: True
 *cupsFilter: "application/vnd.cups-raster 0 hpcups"
-*% Additional support for dj3600command
+*% Additional support for commandtolidil
 *cupsCommands: "Clean PrintAlignmentPage PrintSelfTestPage ReportLevels"
-*cupsFilter: "application/vnd.cups-command 0 dj3600command"
+*cupsFilter: "application/vnd.cups-command 0 commandtolidil"
 *cupsLanguages: "en"
 *UIConstraints: *MediaType Glossy *OutputMode DraftRGB
 *UIConstraints: *OutputMode DraftRGB *MediaType Glossy
